vllm/engine/llm_engine.py

```diff
@@ -1098,6 +1161,7 @@ def _advance_to_next_step(
                 assert len(seq_group.seqs) == 1
                 seq = seq_group.seqs[0]
                 seq.append_token_id(sample.output_token, sample.logprobs)
+                seq_group.update_num_computed_tokens(1)
```

Comment:
Why we need this? Is this a bug before this PR?

Reply:
I believe it was a bug before. I think it was benign, as `num_computed_tokens` is rarely used once the sequence turns into decode. like in https://github.com/vllm-project/vllm/blob/9e5ec35b1f8239453b1aaab28e7a02307db4ab1f/vllm/sequence.py#L576
Let me confirm this.

IIRC, I added this as a sanity check function in scheduler.py (which is now removed) was failing.  This is likely not required, but it'd be a bug without this update.